internal/cli/cmd/install.go

```diff
@@ -262,6 +263,10 @@ cilium install --context kind-cluster1 --helm-set cluster.id=1 --helm-set cluste
 `,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			params.Namespace = namespace
+			// Don't log anything if it's a dry run so that the dry run output can easily be piped to other commands.
+			if params.DryRun || params.DryRunHelmValues {
+				params.Writer = io.Discard
```

Comment:
How about `io.Stderr`?

Reply:
yeah that's another option. still a bit too noisy for my taste 👀